driver/portfolio_runner.py

```diff
@@ -83,9 +83,10 @@ def compute_run_time(timeout, configs, pos):
     print("remaining time: {}".format(remaining_time))
     relative_time = configs[pos][0]
     remaining_relative_time = sum(config[0] for config in configs[pos:])
-    print("config {}: relative time {}, remaining {}".format(
-          pos, relative_time, remaining_relative_time))
-    return limits.round_time_limit(remaining_time * relative_time / remaining_relative_time)
+    absolute_time_limit = limits.round_time_limit(remaining_time * relative_time / remaining_relative_time)
+    print("config {}: relative time {}, remaining time {}, absolute time {}".format(
+          pos, relative_time, remaining_relative_time, absolute_time_limit))
+    return absolute_time_limit
```

Comment:
Print absolute time.

Reply:
This might break some existing parsing scripts, so I'd warn about this in the commit message.